src/lib/components/CommunitySelectionModal/CommunityListItem.js

```diff
@@ -0,0 +1,94 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2022 CERN.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from 'react';
+import { Field } from 'formik';
+import { Item, Button, Container } from 'semantic-ui-react';
+import _truncate from 'lodash/truncate';
+import { Image } from 'react-invenio-forms';
+import PropTypes from 'prop-types';
+import { i18next } from '@translations/i18next';
+
+export const CommunityListItem = ({ result, standAlone }) => {
+  const metadata = result.metadata;
+
+  return (
+    <Field>
+      {({ form: { values, setFieldValue } }) => (
+        <Item key={result.id} className="community-list-result-item">
+          <Item.Image
+            as={Image}
+            size="tiny"
+            src="/static/images/square-placeholder.png"
+          />
+
+          <Item.Content verticalAlign="top">
```

Comment:
can we have each community clickable, and on click, it will open it in a new tab? In this way the user can make sure (s)he will select the right community.

Reply:
that is a good suggestion, will see how that can be implemented :+1: